content/_code-samples/build-a-wallet/js/3_account.js

```diff
@@ -0,0 +1,106 @@
+const { app, BrowserWindow } = require('electron')
+const path = require('path')
+const xrpl = require("xrpl")
+
+const testnetUrl = "wss://s.altnet.rippletest.net:51233"
+
+let reserveBaseXrp = null, reserveIncrementXrp = null
+
+const fixedAddress = 'rn95xwUymaMyzAKnZUGuynjZ6qk9RzV4Q7' // Operational Account Alex
```

Comment:
Will the fixed account work over time? (I'd imagine the amounts could go to zero as people use the final version of the wallet that can actually send transactions or if testnet history is reset) - Would it make sense to create a small script to populate a test account and have it run as part of these versions? (E.g. Create an account then send two payments or something?)

Reply:
Accounts and Seeds are now queried via a dialog, hardcoded stuff has been removed from the code.